roamer/main.py

```diff
@@ -5,24 +5,40 @@
 
 from __future__ import print_function
 import sys
+
+import click
+
 import roamer
 from roamer.session import Session
 
-def start(argv):
-    skipapproval = False
-    path = None
-    if '--path' in argv:
-        i = argv.index('--path') + 1
-        path = argv[i]
-    if '--version' in argv:
+
+@click.command()
+@click.option("--path", type=click.Path(file_okay=False, exists=True), default=None)
+@click.option("--raw-out/--no-raw-out", default=False)
+@click.option("--raw-in/--no-raw-in", default=False)
+@click.option("--skip-approval/--no-skip-approval", "skipapproval", default=False)
+@click.option("--version/--no-version", default=False)
+def start(path, raw_out, raw_in, skipapproval, version):
+    if version:
         print(roamer.__version__)
         return
-    if '--raw-out' in argv:
+
+    # TODO: Is this a bug? If --raw-out/in is specified, then --skip-approval is ignored.
+    # Is that correct? For now, to remain backwards compatible I am maintaining this behaviour.
```

Comment:
This raises a good point. I only envisioned --raw-out/in being used by text editor plugins ([roamer.vim](https://github.com/abaldwin88/roamer.vim)) and thus wasn't too concerned about the confirmation.  The implementer of the plugin would probably want to handle that themselves.

Having said that I think it's best to keep the behavior consistent. Can you update this PR so that `--skip-approval` is never ignored?